### DISCUSSION TITLE
Does Halo Custom Edition work with this mod?

### DISCUSSION THREAD

**Discussion by FelicityFlamel** [LivingFray/HaloCEVR#32] [Q&A]
I just wanted to double check and confirm only the 2003 PC edition will work for this?

**Comment by kitaaaaaa** [comment]
If you are using Chimera, Custom Edition support can be enabled in the retail release of Halo PC by copying bitmaps.map, loc.map, and sounds.map from an installed copy of Halo Custom Edition, appending "custom_" to the beginning of the filename for each, and dropping them in the maps folder of Halo PC. 
This works for both multiplayer and single player maps, although you may need a custom UI.map such as Universal UI in order to access the SP maps easily because the console (~) doesn't render in VR.
